packages/user/src/UserService.ts

```diff
@@ -1,17 +1,17 @@
-import { IUser, IWishCard, ObjectId, TypeObjectId, UserRoles } from '@donategifts/common';
-import { AgencyRepository } from '@donategifts/agency';
-import { WishCardRepository } from '@donategifts/wishcard';
-import { inject, injectable } from 'inversify';
+import { injectable } from 'inversify';
 import * as uuidV4 from 'uuid';
 import * as moment from 'moment';
+import { IUser, IWishCard, ObjectId, TypeObjectId, UserRoles } from '@donategifts/common';
+import { AgencyRepository } from '@donategifts/agency-data';
+import { WishCardRepository } from '@donategifts/wishcard';
 import { sendPasswordResetMail } from '@donategifts/helper';
-import { UserRepository } from './database/UserRepository';
+import { UserRepository } from '@donategifts/user-data';
 import { UserError } from './helper/UserError';
 
 @injectable()
 export class UserService {
 	constructor(
-		@inject(UserRepository) private userRepository: UserRepository,
+		private userRepository: typeof UserRepository = UserRepository,
```

Comment:
Okay, qn about the whole injectable thing - My understanding is that we are injecting to get the single instance and ensure we are using that instance everywhere.....but I suppose in this case it doesn't matter since it's Repository stuff, right? :thinking: But kinda makes default scope in user data pointless, no?

Or does it automatically get the instance without using @ inject?

Reply:
it does without because the userrepository is already bound to a container
you would only need to specifically inject it if it wasn't bound to the container instance, but in the case of using it in the constructor from another package it get's bound in the origin package
![Capture](https://user-images.githubusercontent.com/44695100/104614056-2c144900-5688-11eb-8cb5-e9f5687349e3.PNG)